source/network/ODServer.cpp

```diff
@@ -606,6 +610,7 @@ bool ODServer::processClientNotifications(ODSocketClient* clientSocket)
             const std::string& nick = clientSocket->getPlayer()->getNick();
             int32_t id = clientSocket->getPlayer()->getId();
             int seatId = seat->getId();
+            seat->setMapSize(gameMap->getMapSizeX(), gameMap->getMapSizeY());
```

Comment:
Arf, nevermind. Short contextual diffs are sometimes evil for that sort of things.